tools/packager/CMakeLists.txt

```diff
@@ -36,7 +36,11 @@ FIND_PYTHON_PKG("jsonschema" SEND_ERROR)
 set(PKG_COMMON_FILES
     afu.py gbs.py utils.py metadata/constants.py metadata/__init__.py metadata/metadata.py schema/afu_schema_v01.json)
 
-set(PKG_PYTHON_FILES packager.py ${PKG_COMMON_FILES} README)
+################# generate the py file for the packager ########################
+configure_file(packager.py.in packager.py @ONLY)
```

Comment:
Would it make sense to use the explicit destination path and prefix packager.py with `${CMAKE_CURRENT_BINARY_DIR}/` for clarity? We do it in line 43

Reply:
The python zipper needs the package name and will fail with a full path.  That's the reason for the machinations with the path below.